presto-docs/src/main/sphinx/presto_cpp/features.rst

```diff
@@ -395,6 +395,14 @@ Native Execution only. Enable row number spilling on native engine.
 
 Native Execution only. Enable simplified path in expression evaluation.
 
+``native_expression.max_array_size_in_reduce``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``100000``
+
+Native Execution only. Reduce() function will throw an error if encountered an array of size greater than this.
```

Comment:
Which reduce function does this refer to? This one? 
https://prestodb.io/docs/current/functions/array.html#reduce-array-T-initialState-S-inputFunction-S-T-S-outputFunction-S-R-R
Suggest adding a link to the function this doc is referring to.

Reply:
@steveburnett 
Yes, that one. Do we need to clarify or add a link in the rst?

Reply:
I would suggest a link. I just reviewed to add a link that I tested and verified in a local doc build.